src/main/resources/static/main.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(
+    function () {
+        getUsersList()
+
+        function addRow(number, username, id) {
+            $("#tbody").append(
+                `<tr>
+                   <th scope="row">${number}</th>
+                   <td>${username}</td>
+                   <td><a href="/logs?id=${id}">logs</a></td>
+                 </tr>
+                `
+            )
+        }
+
+        function getUsersList() {
+            $.ajax({
+                type: 'GET',
+                url: `${host}/users`,
```

Comment:
one more thing I do not like with this approach with the host property, you will need a new configuration for any new environment with the new hostname.